samcli/commands/package/package_context.py

```diff
@@ -67,6 +68,7 @@ def __init__(
         self.metadata = metadata
         self.region = region
         self.profile = profile
+        self.on_deploy = on_deploy
```

Comment:
so this is the special case for package happening as part of deploy?

Reply:
I'm just muting some output that comes on the terminal, afaik.

Reply:
sure, but this feels hacky. May be in the future we will find more general use of this variable.